src/cli.ts

```diff
@@ -2,6 +2,7 @@ import * as core from '@actions/core';
 import * as io from '@actions/io';
 import * as tc from '@actions/tool-cache';
 import {promises as fs} from 'fs';
+import {type Args} from './input.js';
```

Comment:
NIT: tu peux aussi faire comme ça de souvenirs, surtout si tu en as plusieurs.

```suggestion
import type {Args} from './input.js';
```